src/service/video-manager-impl.js

```diff
@@ -498,6 +511,27 @@ class VideoEntry {
       this.logCustomAnalytics_(eventType, data['vars']);
     });
 
+    // TODO(alanorozco): Listeners below could be symmetrical (ie. just one)
+    // if consolidating VideoEvents with VideoAnalyticsEvents,
+    // e.g. `video-` prefix. This also applies to most other handlers that
+    // trigger an analyticsEvent.
+    // (Ensure symmetry between types via unit test.)
```

Comment:
Can we make an issue for this?

Reply:
Just realized this doesn't actually matter, `VideoAnalyticsEvents` is small. I feel like there are other opportunities to simplify analytics but unfortunately this isn't it.